docs/variants/talos_2/other-manuals.md

```diff
@@ -1,4 +1,4 @@
-# Insurgo Dasharo on Talos II - Conference materails
+# Other manuals
```

Comment:
We should roll back the name of this file - `other manuals` is a collective term.

Reply:
renamed